src/index.ts

```diff
@@ -0,0 +1,56 @@
+import express = require("express");
+import mongoose = require("mongoose");
+import { ApolloServer } from "apollo-server-express";
+import * as dotenv from "dotenv";
+
```

Comment:
@EhudPer 
maby better to work with logger insted of console.log()

Reply:
The console log is for us during development mainly for now.
What do you suggest?, you want me to add logging to the backend (not instead of the console logs).

Reply:
@EhudPer 
i dont mean loging in, read about logger its a package for showing or storing logs...

Reply:
@Orenlibane So i should add the logger package and change the console logs that we have now to the logger console logs?

Reply:
@skendis the same above question for you too.

Reply:
should add logger service that will write logs to file

Reply:
well, in small scale application it dosent have to be a service, but ye, its a better practice :) 
while in development you want the logs to act as 'console.logs' and while in prod you want them to write to log file.
@EhudPer @skendis

Reply:
`const fs = require('fs')

const logsDir = './logs';
if (!fs.existsSync(logsDir)) {
    fs.mkdirSync(logsDir);
}

//define the time format
function getTime() {
    let now = new Date();
    return now.toUTCString();
}

function doLog(line, level='Debug') {
    line = `${getTime()} - ${level} - ${line}` 
    
    const content = fs.readFileSync('./logs/log.log')
    fs.writeFileSync('./logs/log.log', content + '\n' + line)
    console.log(line);
}

module.exports = {
    debug(line){
        doLog(line, "Debug")
    },
    info(line){
        doLog(line, "Info")
    },
    warn(line){
        doLog(line, "Warn")
    },
    error(line){
        doLog(line, "Error")
    }
}`

should do the job 👍

Reply:
@skendis and @Orenlibane i took Sergei's code and changed it a little.
I think to logger service is now ready and working basically.
You can review it again.